Doc/c-api/frame.rst

```diff
@@ -0,0 +1,75 @@
+.. highlight:: c
+
+Frame Objects
+-------------
+
+.. c:type:: PyFrameObject
+
+   The C structure of the objects used to describe frame objects.
+
+   The structure is only part of the internal C API: fields should not be
```

Comment:
Maybe "The structure is only part of the internal C API" -> "The structure is not part of the C API"